cmd/kube-controller-manager/app/core.go

```diff
@@ -355,13 +355,10 @@ func startGarbageCollectorController(ctx ControllerContext) (bool, error) {
 	restMapper.Reset()
 
 	config := ctx.ClientBuilder.ConfigOrDie("generic-garbage-collector")
```

Comment:
any adjustments to qps/burst needed since you're switching from a pool to a single client?

Reply:
> any adjustments to qps/burst needed since you're switching from a pool to a single client?

ah, probably.  anyone want to guess as to the number of resources we had?  logically this would be their multiplier or we can just guess at 20x normal.